lib/LoadingOverlay/styles.scss

```diff
@@ -51,6 +51,15 @@ $loading-line-width: 32px;
   }
 }
 
+.loading-overlay__content {
```

Comment:
💚 I think we have helper classes for these (except `text-align: center;`? Should be easy to add).

Would be nice to get rid of the `p:last-child` selector, so we wouldn't need to change the CSS if we add another `p` tag